packages/flutter/test/material/slider_test.dart

```diff
@@ -4548,4 +4548,60 @@ void main() {
       paints..scale()..path(color: theme.colorScheme.primary),
     );
   }, variant: TargetPlatformVariant.desktop());
+
+  testWidgets('Value indicator label is shown when focused', (WidgetTester tester) async {
+    double value = 0.5;
+    final FocusNode focusNode = FocusNode();
+    addTearDown(focusNode.dispose);
+
+    Widget buildApp({bool enabled = true}) {
```

Comment:
nit: maybe we can remove the parameter here since we only tested the enable slider:)

Reply:
Done, thanks for the recs!